### ISSUE TITLE
Ambiguous language

### ISSUE THREAD

**Issue by TomArena** [MicrosoftDocs/powerbi-docs#4726]
"Importing dataflows that contain linked tables doesn't fix the existing references within the dataflow (these queries should be fixed manually before importing the dataflow)." 

This usage of "fixed" is ambiguous. Does "fix" in this context mean that the references aren't locked or that they aren't repaired when the Flow is imported? 

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: b3ec2a31-ae63-38f7-00e8-234bcedd571d
* Version Independent ID: ff4d4e25-80f5-cac3-49c8-a85b7cf1c476
* Content: [Dataflows Limitations, restrictions and supported connectors and features - Power BI](https://learn.microsoft.com/en-us/power-bi/transform-model/dataflows/dataflows-features-limitations)
* Content Source: [powerbi-docs/transform-model/dataflows/dataflows-features-limitations.md](https://github.com/MicrosoftDocs/powerbi-docs/blob/main/powerbi-docs/transform-model/dataflows/dataflows-features-limitations.md)
* Service: **powerbi**
* Sub-service: **pbi-dataflows**
* GitHub Login: @davidiseminger
* Microsoft Alias: **davidi**

**Comment by bishalgoswami** [comment]
Thanks for your comment, @TomArena. This issue has been assigned to the doc author for review.

**Comment by davidiseminger** [comment]
Good point, thanks for your feedback. I've improved that to say "update" instead of fix.
